src/controllers/update/beneficial.owner.type.controller.ts

```diff
@@ -0,0 +1,30 @@
+import { NextFunction, Request, Response } from "express";
+
+import { logger } from "../../utils/logger";
+
+import * as config from "../../config";
```

Comment:
minor - import 3 consts instead of *?

Reply:
not changed as moved code into the controller as it did not need to be separated and now adds more than 3 imports